cmd/wharf/root.go

```diff
@@ -42,15 +44,24 @@ var rootCmd = &cobra.Command{
 }
 
 var rootContext, rootCancel = context.WithCancel(context.Background())
+var rootConfig config.Config
 
-func addKubernetesFlags(flagSet *pflag.FlagSet, overrides *clientcmd.ConfigOverrides) {
-	overrideFlags := clientcmd.RecommendedConfigOverrideFlags("k8s-")
-	clientcmd.BindOverrideFlags(overrides, flagSet, overrideFlags)
+var k8sOverridesFlags clientcmd.ConfigOverrides
+
+var runAfterConfig []func()
```

Comment:
Suggest grouping these into a single `var` declaration to make it look prettier :)

GitHub doesn't allow me to add code suggestion, but something like this:

```go
var (
	rootContext, rootCancel = context.WithCancel(context.Background())

	k8sOverridesFlags clientcmd.ConfigOverrides

	rootConfig     config.Config
	runAfterConfig []func()
)
```